app/controllers/abrigos.js

```diff
@@ -21,11 +23,12 @@ exports.create = async (req, res) => {
     foto3: req.body.foto3
   };
 
-
-
+  const senhaCriptografada = await bcrypt.hash(abrigo_params.senha, 10); // gera o hash da senha
+  abrigo_params.senha = senhaCriptografada; // substitui a senha pela sua versão criptografada
```

Comment:
Comentários vão conta as boas práticas do clean code. Se as variaveis do seu código estão bem nomeadas, não eh preciso um comentário para explicá-lo xD xD xD

E no a nomenclatura delas está ótima

```suggestion
  const senhaCriptografada = await bcrypt.hash(abrigo_params.senha, 10);
  abrigo_params.senha = senhaCriptografada; 
```